Makefile

```diff
@@ -66,7 +63,6 @@ uninstall:
 	rm $(GOBIN)/$(BINARY)
 	rm -rf $(HOME)/.edgex-cli
 
-
 clean:
 	-rm $(BINARY)*
```

Comment:
handle the bin directory?

Reply:
I have made clean dependent on uninstall target. (clean: uninstall) This means that clean calls firstly uninstall and proceed